### PR TITLE
[PS-537] Fix Azure build Pipeline

### PR DESCRIPTION
**Pull request by pivnicek**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -1,18 +1,53 @@
 pool:
   vmImage: ubuntu-latest
 
+variables:
+  - name: imageTag
+    value: ${{ replace(variables['Build.SourceBranch'],'refs/tags/','') }}
+  - name: isTagBuild
```
Comment (JakubMatejka) [inline azure-pipelines.yml:7]: This variable means that the commit is tagged, right? Then I would maybe rather called it `isTaggedBuild` or `isBuildTagged` just to be more explainable.

**Review by JakubMatejka — COMMENTED** [review]
I'm not sure I'm the one who should review it, I just guess that it could work. 😄

**Comment by pivnicek** [comment]
yeah, I was thinking it may be a good idea to go over the build with you in a short call at some point

**Review by JakubMatejka — APPROVED** [review]
Ok, I think I know what's supposed to happen. (And it won't break anything anyway, right? 😄 )

**Comment by pivnicek** [comment]
yeah, it won't break anything :-)

**Pull request merged**